src/prommis/solvent_extraction/sx_model_flowsheet_dynamic.py

```diff
@@ -0,0 +1,232 @@
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Sc"] = 99.1 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Y"] = 99.9 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "La"] = 32.4 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Ce"] = 58.2 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Pr"] = 58.2 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Nd"] = 87.6 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Sm"] = 99.9 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Gd"] = 69.8 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Dy"] = 96.6 / 100
+    else:
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Al"] = 4.9 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Ca"] = 12.3 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Fe"] = 6.4 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Sc"] = 16.7 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Y"] = 99.9 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "La"] = 23.2 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Ce"] = 24.9 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Pr"] = 15.1 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Nd"] = 99.9 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Sm"] = 99.9 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Gd"] = 7.6 / 100
+        m.fs.solex.partition_coefficient[s, "aqueous", "organic", "Dy"] = 5 / 100
+
+
+"""
+Fixation of the inlet conditions and the initial state values for all the components.
+
+"""
+
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "H2O"].fix(1e6)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "H"].fix(1.755)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "SO4"].fix(3999.818)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "HSO4"].fix(693.459)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "Al"].fix(422.375)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "Ca"].fix(109.542)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "Cl"].fix(1e-7)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "Fe"].fix(688.266)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "Sc"].fix(0.032)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "Y"].fix(0.124)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "La"].fix(0.986)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "Ce"].fix(2.277)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "Pr"].fix(0.303)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "Nd"].fix(0.946)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "Sm"].fix(0.097)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "Gd"].fix(0.2584)
+m.fs.solex.aqueous_inlet.conc_mass_comp[:, "Dy"].fix(0.047)
```

Comment:
Where do these values comes from?

Reply:
It was taken quite some time back from the git prommis leaching flowsheet product stream values.